src/systems/filecoin_nodes/key_store/key_store.id

```diff
@@ -0,0 +1,15 @@
+import filcrypto "github.com/filecoin-project/specs/algorithms/crypto"
```

Comment:
see below @whyrusleeping @jbenet 

I would argue we may want to disable the ability to have many workers per owner in v1.

We do need this abstraction if we want to enable key changes though (I can spec in forthcoming PR)

Reply:
>  I would argue we may want to disable the ability to have many workers per owner in v1.

yeah me too, I remember that the rationals were not clearly expressed last time we discussed this and were leaning towards being opinions rather than fact-based decisions